### DISCUSSION TITLE
Git extensions Git commit message

### DISCUSSION THREAD

**Discussion by mfarghaly9** [gitextensions/gitextensions#10012] [Q&A]
Each time i commit a change on git 
git extensions appends my commit message with change-id-some number 
what does it refer to and can i Block that from being generated

**Comment by pmiossec** [comment]
Can you add a screen shot because we don't know what you are talking about (and GitExtensions or git should not change the git message)? 

It seems more like a git hook that you have set up....

**Reply by mfarghaly9** [reply]
![Capture](https://user-images.githubusercontent.com/101417157/170721903-4a1e5700-bf97-41a8-918c-4d9b83ea2c3f.JPG)
This is the issue

**Reply by RussKie** [reply]
The app doesn't add anything like this.

**Reply by pmiossec** [reply]
> ![Capture](https://user-images.githubusercontent.com/101417157/170721903-4a1e5700-bf97-41a8-918c-4d9b83ea2c3f.JPG) This is the issue

Do you use Gerrit because a quick search seems to show that it's something added by gerrit:
https://gerrit-review.googlesource.com/Documentation/user-changeid.html
